utils/docker/Dockerfile.centos.8

```diff
@@ -48,13 +97,15 @@ RUN dnf clean all && dnf -y install epel-release dnf-plugins-core && \
         gcc \
         gcc-c++ \
         git \
+        glibc-langpack-en \
         golang \
         graphviz \
         hwloc-devel \
         ipmctl \
         java-1.8.0-openjdk \
         json-c-devel \
         libaio-devel \
+        libatomic \
```

Comment:
Are you sure this is needed?  As I understand this is apparently an alternative to openpa, which mercury wants.

However mercury builds on EL8 just fine without either libatomic or openpa.

And we are already building DAOS on EL8 without this in the Dockerfile so why is it needed now?

Reply:
Yes.
Developers are building DAOS to use libatomic instead of openpa.
If libatomic is not present, the build will fall back to using openpa.

Once we get everything building with libatomic, we can stop building and packaging openpa.   CentOS 7 was the last distro to provide libatomic.

Next set of CI images will have libatomic pre-loaded.

There is a ticket to have the dependency of openpa totally removed from DAOS.

Reply:
Did you miss this from my original comment: _mercury builds on EL8 just fine without either libatomic or openpa_.

Reply:
No I did not miss it.

If scons detects libatomic, it builds with libatomic, otherwise it falls back to openpa, building it if needed.

On CentOS 8, it is likely that some other package needed or supplied with the base OS is causing the libatomic package to be present, or there is some other reason that it does not need to be explicitly installed, because otherwise scons would be building openpa and mercury, etc, would end up dependent on it.

Reply:
This [build](https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fmercury/detail/PR-56/3/pipeline) of mercury demonstrates that neither [openpa nor libatomic are installed](https://build.hpdd.intel.com/job/daos-stack/job/mercury/job/PR-56/3/artifact/artifacts/centos8/root.log) into the chroot for the build of mercury.  The CentOS 7 build of the same PR does indeed demonstrate that openpa is needed if libatomic is not installed.

Interestingly the Leap 15 build works without openpa as it installs libatomic1 as part of the `patterns-devel-base-devel_rpm_build` package install.

Reply:
Are you sure that nothing in the chroot is causing libatomic or equivalent to be installed?

According to the log for the PR, it is being installed, so it is not present in the Docker image, but there may be an alternate package that the chroot is using.

I have not looked at specifically what scons is looking for to prevent openpa from being built.

So for now, want to be defensive and make sure that libatomic is present.

Reply:
> Are you sure that nothing in the chroot is causing libatomic or equivalent to be installed?

Yes.  I examined the root.log the link to which I included in my previous comment.

> According to the log for the PR, it is being installed

Which log exactly?

> so it is not present in the Docker image, but there may be an alternate package that the chroot is using.

Again, no.  I examined the root.log, again which I provided a link to in my previous comment.  Take a look for yourself.  There is no `openpa` nor `libatomic` being installed.

There is NO openpa build for EL8.  There won't be.  You can remove libatomic from the EL8 Dockerfile and see for yourself that mercury still builds.  If it builds, it's fine.  Mercury fails out of it's `cmake` with an error if a suitable atomic cannot be found.

Reply:
The rpmbuild of mercury on CentOS 8 has the libatomics_ops-7.6.2 package installed.
Assuming what ever pulled that in is present for CentOS 8 builds.

Reply:
Are you sure libatomic_ops and libatomic are the same thing?